### PR TITLE
backport getOrgId fix

### PR DESCRIPTION
**Pull request by GeekSheikh**
backporting critical fix applied in 0720

### CODE REVIEW

File: src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala [hunk 1]
```diff
@@ -561,10 +561,38 @@ object Initializer extends SparkSessionWrapper {
...
+  def getOrgId: String = getOrgId(None)
+
+  /**
+   * Returns the local workspace orgID, or the orgID contained in the input apiURL
+   * When getOrgId is called from a Future the apiURL MUST be passed to prevent failure to get orgID
+   * since dbutils.notebook.getContext.apiUrl.get returns None and None.get fails
+   * @param apiUrl apiURL of the workspace to get the orgId
+   * @return
+   */
+  def getOrgId(apiUrlInput: Option[String]): String = {
```
Comment (gueniai) [inline src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala:587]: We should also call this one from `Initializer.initConfigState`

Comment (gueniai) [reply src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala:587]: also from `Helpers.getWorkspaceByDatabase`

**Review by gueniai — CHANGES_REQUESTED** [review]
There are 2 other places where this function is called that we should backport as well.

File: src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala (hunk 1 above)
Comment (GeekSheikh) [reply src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala:587]: nice catch, thank you!

**Review by gueniai — APPROVED** [review]
ship it!

**Pull request merged**